### PR TITLE
fix: gossipsub seen_ttl for EIP-7045

### LINKED ISSUE #6567: Extend `ttl_seen` valid lifetime of an attestation/aggregate to 2 epochs
### Problem description

We've also confirmed that we are seeing spam post-Deneb under the aggregate channel.

photo_2024-03-19 09 53 37

### Solution description

> Then net effect of the above change is that peers are allowed to republish old aggregates and attestations and libp2p will not stop the spread with the settings we recommend - instead the messages will have to be stopped with the "attestation cover rule" or similar, even though they have been observed already.

### Additional context

Reference: https://github.com/ethereum/consensus-specs/pull/3627

### PR DESCRIPTION
**Pull request by twoeths**
**Motivation**

See [6567](https://github.com/ethereum/consensus-specs/pull/3627)

**Description**

Closes #6567

### CODE REVIEW

File: packages/beacon-node/src/network/gossip/gossipsub.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class Eth2Gossipsub extends GossipSub {
       fanoutTTL: 60 * 1000,
       mcacheLength: 6,
       mcacheGossip: 3,
-      seenTTL: 550 * GOSSIPSUB_HEARTBEAT_INTERVAL,
+      seenTTL: config.SECONDS_PER_SLOT * SLOTS_PER_EPOCH * 2,
```
Comment (jeluard) [inline packages/beacon-node/src/network/gossip/gossipsub.ts:102]: Would adding a comment pointing to the specs make sense here?

**Review by jeluard — APPROVED** [review]

File: packages/beacon-node/src/network/gossip/gossipsub.ts (hunk 1 above)
Comment (twoeths) [reply packages/beacon-node/src/network/gossip/gossipsub.ts:102]: this is a pretty straightforward value which can be easily found in the spec
also EIP-7045 happens in a couple of places not just this one

**Comment by twoeths** [comment]
Some other clients already merged this updated `seenTTL`

File: packages/beacon-node/src/network/gossip/gossipsub.ts (hunk 1 above)
Comment (twoeths) [reply packages/beacon-node/src/network/gossip/gossipsub.ts:102]: this should be in `ms` not `second`, fixed in #6607

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.18.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.18.0) :tada:

**Pull request merged**